analysis/deathknightunholy/src/integrationTests/example.test.ts.snap

```diff
@@ -4424,7 +4424,50 @@ exports[`Unholy Death Knight integration test: example log InscrutableQuantumDev
                   onTouchStart={[Function]}
                   style={Object {}}
                 >
-                  100
+                  50
+                  %
+                </dfn>
+              </div>
+            </div>
+            <div
+              className="flex"
+            >
+              <div
+                className="flex-sub"
+              >
+                <div
+                  className="circle"
+                  style={
+                    Object {
+                      "background": "hotpink",
+                    }
+                  }
+                />
+              </div>
+              <div
+                className="flex-main"
+              >
+                <dfn
+                  className=""
+                  onMouseEnter={[Function]}
+                  onMouseLeave={[Function]}
+                  onTouchStart={[Function]}
+                  style={Object {}}
+                >
+                  ???
```

Comment:
I'm not sure why the integration tests ends up with more casts than detected events.. Can the logs in the zips differ from current WCL logs? Can I view the logs that was used to create the tests somehow? 🤔

I hope it's just failing the decoy tracking, which should happen so damn rarely that it's fine, but I'm not sure.